src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -620,27 +625,26 @@ public JSONArray validateRequiredValues(MarcReader reader,String token, String b
 			    if (orgValidationResult != null) {
 			    	logger.error("organization invalid: "+ vendorCode);
 			    	logger.error(record.toString());
-			    	responseMessages.put(orgValidationResult);				    
+			    	errorMessages.put(orgValidationResult);				    
 			    }
 			    				    
 			    JSONObject fundValidationResult = validateFund(fundCode, title, token, baseOkapEndpoint, price);
 			    if (fundValidationResult != null) {
 			    	logger.error("fundCode invalid: "+ fundCode + " (price: "+ price +")");
 			    	logger.error(record.toString());
-			    	responseMessages.put(fundValidationResult);
+			    	errorMessages.put(fundValidationResult);
 			    }
-			    return responseMessages;
-			    
 			}  catch(Exception e) {
 				e.printStackTrace();
 		    	logger.fatal(e.getMessage());
-		    	JSONObject responseMessage = new JSONObject();
-		    	responseMessage.put("error", e.getMessage());
-		    	responseMessage.put("PONumber", "~error~");
-		    	responseMessages.put(responseMessage);
+		    	JSONObject errorMessage = new JSONObject();
+		    	errorMessage.put("error", e.getMessage());
+		    	errorMessage.put("PONumber", "~error~");
+		    	errorMessages.put(errorMessage);
+                return errorMessages;
```

Comment:
Just fixing the indent...

```suggestion
		    	return errorMessages;
```

Reply:
If it gets to line 644 there were no errors encountered.  It "could" return errorMessages or just an empty JSONArray.  I chose to do the latter because it suggests that there were no validation errors.  

Indenting change is fine....indenting is pretty much a mess on all the code

Reply:
I agree on the formatting. Maybe we can consider adding a linter check to the GA workflow as well after you introduce the unit tests.